recipes/nm.rb

```diff
@@ -227,5 +227,15 @@
   end
 end
 
+# Register NodeManager with Consul
+template "#{node['hops']['bin_dir']}/consul/nm-health.sh" do
+  source "consul/nm-health.sh.erb"
+  owner node['hops']['rm']['user']
```

Comment:
Why is the owner the `rm` user and not `yarn`?

Reply:
Fixed, owner is `yarn` now